lib/post/pages/post_page.dart

```diff
@@ -188,7 +190,7 @@ class _PostPageState extends State<PostPage> {
                                       : singlePressAction == PostFabAction.changeSort
                                           ? () => showSortBottomSheet(context, state)
                                           : singlePressAction == PostFabAction.replyToPost
-                                              ? replyToPost
+                                              ? () => replyToPost(context)
```

Comment:
This should already be in `develop`, so once you update your branch, hopefully there will be fewer changes here.

https://github.com/thunder-app/thunder/blob/develop/lib/post/pages/post_page.dart#L191